src/app/login/Login.tsx

```diff
@@ -44,7 +44,7 @@ export const Login = () => {
                 <p>Redirecting...</p>
               ) : showDisclaimer ? (
                 <Disclaimer
-                  onConnect={() => router.push('/user')}
+                  onConnect={() => router.push('/user?tab=holdings')}
```

Comment:
Are we sure that this doesn't break when the user connected isn't a Builder?

Reply:
I don't think I did test that, no. don't see why it should though, as it just renders the page without the tabs

Reply:
if that's what it does, it's fine 👍🏼